src/pretix/presale/templates/pretixpresale/event/order_change_confirm.html

```diff
@@ -0,0 +1,219 @@
+{% extends "pretixpresale/event/base.html" %}
+{% load i18n %}
+{% load classname %}
+{% load eventurl %}
+{% load money %}
+{% block title %}{% blocktrans trimmed with code=order.code %}
+    Change order: {{ code }}
+{% endblocktrans %}{% endblock %}
+{% block content %}
+    <h2>
+        {% blocktrans trimmed with code=order.code %}
+            Change order: {{ code }}
+        {% endblocktrans %}
+    </h2>
+
+    <form method="post" class="form-horizontal" href="">
+        {% csrf_token %}
+
+        <p>{% trans "Please confirm the following changes to your order." %}</p>
+        <div class="row-fluid">
+            <div class="panel panel-primary">
+                <div class="panel-heading">
+                    <h3 class="panel-title">
+                        {% trans "Change summary" %}
+                    </h3>
+                </div>
+                <table class="panel-body table table-hover">
+                    {% for op in operations %}
+                        {% if op|classname == "ItemOperation" %}
```

Comment:
I see you do string-based class-name comparison to differentiate between change-operations. Usually I’d go with `isinstance`, but due to the string-based nature of params in template-filters, this is probably the best way. Of course it does not test for inheritance, but that is probably not needed.

Or would something like `{% if op is ItemOperation %}` work? I did not get it to work locally, but might be missing something.

Reply:
No, ``is`` checks for object identity, not class hierarchy. Since these operation classes are namedtuples, inheritance is not really relevant.

Reply:
Thanks you for clarifying!